src/pages/index.js

```diff
@@ -93,26 +90,102 @@ const cardForm = cardModal.querySelector(".modal__form");
 const cardCaptionInput = cardModal.querySelector("#card-caption-input");
 const cardLinkInput = cardModal.querySelector("#card-link-input");
 const cardSubmitButton = cardModal.querySelector(".modal__submit-button");
+
+let selectedCard;
+let selectedCardId;
+
+//preview modal elements
 const previewModal = document.querySelector("#preview-modal");
 const previewModalImageElement = document.querySelector(".modal__image");
 const previewModalCaption = document.querySelector(".modal__caption");
 const previewModalCloseButton = document.querySelector(
   ".modal__close-button_type_preview"
 );
 
+//avatar modal elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarModalButton = document.querySelector(
+  ".profile__avatar_edit-button"
+);
+const avatarModalCloseButton = avatarModal.querySelector(
+  ".modal__close-button"
+);
+const avatarSubmitButton = avatarModal.querySelector(".modal__submit-button");
+const avatarLinkInput = avatarModal.querySelector("#avatar-link-input");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarImage = document.querySelector(".profile__avatar");
+
+//delete modal elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".modal__form");
+const deleteModalSubmitButton = deleteModal.querySelector(
+  ".modal__delete-button"
+);
+const deleteModalCancelButton = document.querySelector("#cancel-button");
+const deleteModalCloseButton = deleteModal.querySelector(
+  ".modal___close-button_type_delete"
+);
+
 function handleCardSubmit(evt) {
   evt.preventDefault();
-  console.log(cardLinkInput.value);
-  console.log(cardCaptionInput.value);
-  const inputValues = {
-    name: cardCaptionInput.value,
-    link: cardLinkInput.value,
-  };
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  evt.target.reset();
-  disableButton(cardSubmitButton, validationConfig); //settings
-  closeModal(cardModal);
+  const submitButton = evt.submitter;
+  console.log(submitButton);
+  setButtonText(submitButton, true, "Saving...", "Save");
+  api
+    .postNewCard({
+      name: cardCaptionInput.value,
+      link: cardLinkInput.value,
+    })
+    .then((data) => {
+      cardCaptionInput.textContent = data.name;
+      cardLinkInput.src = data.link;
```

Comment:
Please, delete them. You don't need to set the `textContent` and `src` for the **inputs**